scripts/docker_startup.sh

```diff
@@ -2,6 +2,7 @@
 #add files to build here
 
 cd /app/src/cpp
-emcc -lembind -o file_handler.js file_handler.cpp -s EXPORT_ES6=1 -s MODULARIZE=1 -s NO_DISABLE_EXCEPTION_CATCHING
+emcc -lembind -o Calculator.js Calculator.cpp -s EXPORT_ES6=1 -s MODULARIZE=1 -s ENVIRONMENT="web" -s USE_ES6_IMPORT_META=0 -s SINGLE_FILE
+emcc -lembind -o file_handler.js file_handler.cpp -s EXPORT_ES6=1 -s MODULARIZE=1 -s ENVIRONMENT="web" -s USE_ES6_IMPORT_META=0 -s SINGLE_FILE
```

Comment:
The flags `-s USE_ES6_IMPORT_META=0 -s SINGLE_FILE` should probably not be used when developing (only for tests) since they won't be used in production.

The flag `-s NO_DISABLE_EXCEPTION_CATCHING` enables exception catching of C++ code which helps a lot during development and should be used.